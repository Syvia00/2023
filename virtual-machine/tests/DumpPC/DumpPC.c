int volatile *const DumpPC = (int *)0x0820;
int main() {
    *((char *)0x0800) = 'a';
    *DumpPC;
    return 0;
}
