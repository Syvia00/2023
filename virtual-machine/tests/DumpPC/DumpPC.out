a12CPU halt requested